docs/development/index.md

```diff
@@ -20,39 +18,29 @@ pynetbox components are arranged into modules:
 * `tests/` - Test suite including unit and integration tests
 * `docs/` - Documentation files
 
-Proposing Changes
----------------
+## Proposing Changes
 
 All substantial changes made to the code base are tracked using GitHub issues. Feature requests, bug reports, and similar proposals must all be filed as issues and approved by a maintainer before work begins. This ensures that all changes to the code base are properly documented for future reference.
 
 To submit a new feature request or bug report for pynetbox, select and complete the appropriate issue template. Once your issue has been approved, you're welcome to submit a pull request containing your proposed changes.
 
-.. note::
-   Avoid starting work on a proposal before it has been accepted. Not all proposed changes will be accepted, and we'd hate for you to waste time working on code that might not make it into the project.
+!!! note
+    Avoid starting work on a proposal before it has been accepted. Not all proposed changes will be accepted, and we'd hate for you to waste time working on code that might not make it into the project.
 
-Getting Help
------------
+## Getting Help
 
 There are two primary forums for getting assistance with pynetbox development:
 
-* GitHub discussions - The preferred forum for general discussion and support issues. Ideal for shaping a feature requests prior to submitting an issue.
-* #netbox on NetDev Community Slack - Good for quick chats. Avoid any discussion that might need to be referenced later on, as the chat history is not retained indefinitely.
+* [GitHub discussions](https://github.com/netbox-community/pynetbox/discussions) - The preferred forum for general discussion and support issues. Ideal for shaping a feature requests prior to submitting an issue.
+* [#netbox on NetDev Community Slack](https://netdev.chat) - Good for quick chats. Avoid any discussion that might need to be referenced later on, as the chat history is not retained indefinitely.
 
-.. note::
-   Don't use GitHub issues to ask for help: These are reserved for proposed code changes only.
+!!! note
+    Don't use GitHub issues to ask for help: These are reserved for proposed code changes only.
```

Comment:
Apologies, didn't notice this one before, but this aside is not being rendered correctly.